pkg/gardenlet/controller/shoot/shoot/reconciler_reconcile.go

```diff
@@ -493,6 +493,10 @@ func (r *Reconciler) runReconcileShootFlow(ctx context.Context, o *operation.Ope
 						}
 
 						shoot.Status.EncryptedResources = encryptedResources
```

Comment:
Add a todo in which release this line should be removed (after 3 releases).
```suggestion
                        // TODO(AleksandarSavchev): Stop setting the shoot.Status.EncryptedResources after  v... has been released.
						shoot.Status.EncryptedResources = encryptedResources
```